### DISCUSSION TITLE
Is it possible to have a button set for: [tap, double tap, and hold]?  Giving the button 3 functions.

### DISCUSSION THREAD

**Discussion by Git-Forked** [AntiMicroX/antimicrox#1007] [Q&A] [answered]
There is not enough buttons for each programmable action I would like to have on the controller, so I would like to make each button have more functions.  Thank you.

**Comment by jesusthief** [comment, marked as answer]
I found this page about tap, double tap, and hold. Hope it helps.

https://github.com/AntiMicro/antimicro/wiki/Advanced-Assignments#double-tap
